next-app/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
nit: could you make this PR into two commits, one that is just "stamp out a vanilla next.js app" and the second is "make it work with Bazel" so that second commit can be more instructive

Reply:
👍